lua/hawtkeys/ui.lua

```diff
@@ -192,7 +194,26 @@ M.show_all = function()
         height = height,
         footer = "Current Keybindings",
     })
-    vim.api.nvim_buf_set_lines(ResultBuf, 0, -1, false, ShowAll.show_all())
+    local all = ShowAll.show_all()
+    for i, data in ipairs(all) do
+        local l1 = tostring(data.lhs)
+            .. " ("
+            .. (data.mode or "n")
+            .. ") - "
+            .. tostring(data.from_file):gsub(vim.env.HOME, "~")
+        local l2 = tostring(data.rhs or "<unknown>")
+        vim.api.nvim_buf_set_lines(
+            ResultBuf,
+            i == 1 and 0 or -1,
+            -1,
+            false,
+            { l1 }
+        )
+        -- mapping rhs as extmark so the cursor skips over it
+        vim.api.nvim_buf_set_extmark(ResultBuf, ns, i - 1, 0, {
+            virt_lines = { { { l2, "Comment" } } },
```

Comment:
I dont think I like the "action" being in comment highlighting, could we make this function highlighting and then make the filename in comment highlighting?